netlab_testing/HTTPTests.cpp

```diff
@@ -257,13 +111,13 @@ TEST_F(HTTP_Tests, HTTPS_GET_inet_os) {
 		{"param3", "value3"},
 		{"param4", "value4"}
 	};
-
+	
 	int getRequestResult = http_client->get(get_request_uri, get_request_version, get_headers, get_params);
 	ASSERT_EQ(getRequestResult, RESULT_SUCCESS);
 
 	std::string received_request;
 
-	http_server->client_socket->recv(received_request, 200, 1, 0); 
+	http_server->client_socket->recv(received_request, 131, 0, 0);
```

Comment:
131?

Reply:
Magic number, will be removed. Simply checks that the data serialized and deserialized match.